dist/pathfora.js

```diff
@@ -2621,7 +2651,7 @@
      * @param {object|array}   widgets
      * @param {object}         config
      */
-    this.initializeWidgets = function (widgets, config) {
+    this.initializeWidgets = function (widgets, config, oldConfig) {
```

Comment:
this is fine tho it seems a little clunky, especially if it changes again? another approach might be something like 

```javascript
var sampleargs = function () {
	var widgets, config;

	// in the legacy version of pathfora we allowed for account id to be passed
	// this is now pulled directly from lio to avoid missconfigurations
	if (arguments.length >= 3) {
		widgets = arguments[0];
		config = arguments[2];
	} else {
		widgets = arguments[0];

		if (typeof arguments[1] !== 'string') {
			config = arguments[1];
		}
	}

	console.log('widgets: ' + widgets, '  config: ' +config);
}
```